google/cloud/aiplatform/v1/dataset_client.h

```diff
@@ -242,6 +242,7 @@ class DatasetServiceClient {
   /// [`Status`]: @ref google::cloud::Status
   /// [google.cloud.aiplatform.v1.Dataset]: @googleapis_reference_link{google/cloud/aiplatform/v1/dataset.proto#L36}
   /// [google.cloud.aiplatform.v1.UpdateDatasetRequest]: @googleapis_reference_link{google/cloud/aiplatform/v1/dataset_service.proto#L280}
+  /// [google.protobuf.FieldMask]: @googleapis_reference_link{google/protobuf/field_mask.proto#L242}
```

Comment:
FYI: I think `@googleapis_reference_link{}` does not work for well-known protos.  It will expand to:

https://github.com/googleapis/google-cloud-cpp/blob/6203e23223c5093aee647284fbbc08744a27b02d/cmake/GoogleapisConfig.cmake#L36

And the file does not exist in the `googleapis/googleapis` repository:

https://github.com/googleapis/googleapis/tree/master/google/protobuf

In a separate PR we should make the `ResolveCommentReferences()` function do something clever for well-known protos.